$NetBSD: patch-aa,v 1.11 2011/04/04 13:42:08 wiz Exp $

Bring back DESTDIR support which is somehow patched out by pkgsrc.
    localedir = $(DESTDIR)$(datadir)/locale
is changed to
    localedir = /usr/pkg/share/locale
and the same for gnulocaledir.

--- po/Makefile.in.in.dist	2011-02-27 00:39:47.000000000 +0100
+++ po/Makefile.in.in	2011-02-27 01:11:12.000000000 +0100
@@ -126,8 +126,8 @@
 	for cat in $$catalogs; do \
 	  cat=`basename $$cat`; \
 	  case "$$cat" in \
-	    *.gmo) destdir=$(gnulocaledir);; \
-	    *)     destdir=$(localedir);; \
+	    *.gmo) destdir=$(DESTDIR)$(gnulocaledir);; \
+	    *)     destdir=$(DESTDIR)$(localedir);; \
 	  esac; \
 	  lang=`echo $$cat | sed 's/\$(CATOBJEXT)$$//'`; \
 	  dir=$$destdir/$$lang/LC_MESSAGES; \
